common/src/main/java/com/wynntils/services/itemvault/type/SavedItem.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.itemvault.type;
+
+import java.util.Set;
+
+public record SavedItem(
+        String base64, Set<String> categories, int itemID, int damage, int hideFlags, boolean unbreakable, int color)
```

Comment:
Perhaps you could make an inner class here, that stores all `ItemStack` related  info.